stock_vertical_lift/models/vertical_lift_operation_pick.py

```diff
@@ -70,10 +85,15 @@ def fetch_tray(self):
     def select_next_move_line(self):
         self.ensure_one()
         next_move_line = self.env["stock.move.line"].search(
-            self._domain_move_lines_to_do(), limit=1
+            self._domain_move_lines_to_do(), limit=1, order="vertical_lift_pick_skipped"
```

Comment:
I think we still have to ensure the original order is kept, maybe by concatenating `vertical_lift_pick_skipped` and `self._order`?

Reply:
Done.